webapp/app/components/roomList/roomList.component.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import { ifElse, equals, always } from 'ramda';
+import List, { ListItem, ListItemIcon, ListItemText } from 'material-ui/List';
+import ChatIcon from 'material-ui-icons/Chat';
+
+import { RoomLink } from './roomList.styles';
+
+
+export class RoomList extends PureComponent {
+  static propTypes = {
+    rooms: PropTypes.instanceOf(Map),
+    activeRoom: PropTypes.object.isRequired,
+  };
+
+  renderList = () => {
+    if (this.props.rooms.size) {
+      const sortedRooms = this.props.rooms.sort();
+      const isActive = (id) => ifElse(
```

Comment:
mozesz usunac to przekazywanie id na koncu ifElse

Reply:
dodatkowo wystarczy tutaj when

Reply:
Z `when` prawda, bez `id` na końcu, nie działa :(

Reply:
`when` jednak też nie jest dobry, bo gdy warunek nie jest spełniony przekazywana jest otrzymana wartość, a to powoduje warningi w konsoli 
```
warning.js:33 Warning: Invalid attribute name: `0`
```